contracts/plugins/assets/frax-eth/SFraxEthCollateral.sol

```diff
@@ -51,6 +57,8 @@ contract SFraxEthCollateral is AppreciatingFiatCollateral {
         high = p + err;
         // assert(low <= high); obviously true just by inspection
 
+        // Note: Currently not checking for depegs between `frxETH` and `ETH`
```

Comment:
Can we make this a `TODO`? Can keep the rest of the text, I just like having the string to search for

Reply:
Yeah I first did a TODO but then realized we dont have ANY in any of our contracts. But I can do it.

Reply:
Done